topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: tutorial_hands_on
+topic_name: sequence-analysis
+tutorial_name: "annotation-with-prokka"
+---
+
+# Introduction
+{:.no_toc}
+
+In this section we will use a software tool called Prokka to annotate the draft genome sequence produced in the previous [tutorial](/modules/spades/index.md). Prokka is a “wrapper”; it collects together several pieces of software (from various authors), and so avoids “re-inventing the wheel”.
+
+Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Note, Prokka uses a two-step process for the annotation of protein coding regions: first, protein coding regions on the genome are identified using [Prodigal](http://prodigal.ornl.gov/); second, the *function* of the encoded protein is predicted by similarity to proteins in one of many protein or protein domain databases. Prokka is a software tool that can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats. More information about Prokka can be found [here](https://github.com/tseemann/prokka).
+
+## Learning objectives
+{:.no_toc}
+
+At the end of this tutorial you should be able to:
+
+1. load a genome assembly into Prokka
+2. annotate the assembly using Prokka
+3. examine the annotated genome using JBrowse
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Import the data
+
+Prokka requires assembled contigs.
+
+> ### {% icon hands_on %} Hands-on: Obtaining our data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    > ### {% icon tip %} Starting a new history
+>    >
+>    > * Click the **gear icon** at the top of the history panel
+>    > * Select the option **Create New** from the menu
+>    {: .tip}
+>
+> 2. **Import Sample Data.** The data for this course may be available from a shared library in Galaxy
+> (ask your instructor). If this is not the case, you can upload it yourself.
+> - Option 1: From data library:
```

Comment:
Currently we are trying to avoid any reference to the data library because it can change from one Galaxy instance to the other. We are explaining during the workshop how to do that.
Could you remove it? Thanks!

Reply:
Yes, removed.